server/src/MessageHandler.ts

```diff
@@ -143,7 +143,7 @@ export function handleDefinitionRequest(
 
   const modelDefinition = 'model '
   // get start position of model type
-  const index = documentText.indexOf(modelDefinition + modelName + ' ')
+  const index = documentText.search(modelDefinition + modelName + /\s+/)
```

Comment:
So this won't work if the auto formating is disabled in VSCode?

`Open your Command Palette (CMD/CTRL+SHIFT+P) and type Toggle Format on Save, then hit Enter`

Example 
```prisma
model helloworld{
....
}

Reply:
Good point, I'll have to change that. Unfortunately the function I'd need for that,
String.prototype.matchAll() is part of the ECMAScript 2020 specification (draft)

Reply:
Oh ok, maybe this could help then https://www.npmjs.com/package/string.prototype.matchall

Reply:
Yes, but I also found another mistake, right now it would only work in case there is exactly one white space between `model` and `helloworld`. I used a regexp instead: `new RegExp("model\\s+" + modelName + "[\\s+|{]")`

Reply:
Oh indeed! I'm not sure the + is needed though 🤔 
<img width="599" alt="Screen Shot 2020-05-14 at 13 05 48" src="https://user-images.githubusercontent.com/1328733/81927342-c85eeb00-95e3-11ea-8508-3de6cbe8848f.png">
https://regex101.com/

Reply:
yep, you're right